### ISSUE TITLE
Plugin not enforcing the configured settings

### ISSUE THREAD

**Issue by alexdao3** [avaly/eslint-plugin-import-order#2]
Hey there! Thanks for this plugin. The behavior seems inconsistent, and I wanted to see if you had any ideas/suggestions. I haven't dug into the plugin code yet, but plan to do that at some point.

```
// .eslintrc.js 

'@avaly/import-order/order': [
  'error',
  {
    groups: [
      'builtin',
      'external',
      'internal',
      'parent',
      'sibling',
      'index',
      'flow',
    ],
  },
],

```

```
import type {
  SomeType
} from './types'
import reducer, {
  someAction
} from './store'
import * as API from '@/api'
import objectAssign from 'object-assign'
```

correctly reports that `'object-assign' import should occur before import of './types'`

However, once `object-assign` is moved a line up, the error disappears, and no other errors surface.

```
import type {
  SomeType
} from './types'
import reducer, {
  someAction
} from './store'
import objectAssign from 'object-assign'
import * as API from '@/api'
```

So not only is `object-assign` no longer an issue, but `eslint` does not error on any of the other invalidly-ordered imports

**Comment by avaly** [comment]
I published `v1.0.2` with a fix for this.

You'll need to add `unknown` to the list of groups in your rule config.

This is required due to how the base import plugin detects import types. I assume your `@/api` is an internal import. Unfortunately the rule to detect internal imports is that is that it matches the regexp `/^\w/`, which your import does not. So it ends up being an `unknown` import.

**Comment by alexdao3** [comment]
Appreciate the insight and the quick turnaround! You're correct - `@/` is our internal import.

For my understanding, why would the `unknown` import trick the plugin into thinking everything was ok (even though the `external` was ordered incorrectly?)
